SampleApplications/Samples/NetCoreConsoleServer/Program.cs

```diff
@@ -142,34 +144,36 @@ public static int Main(string[] args)
 
     public class MySampleServer
     {
-        SampleServer server;
-        Task status;
-        DateTime lastEventTime;
-        int serverRunTime = Timeout.Infinite;
-        static bool autoAccept = false;
-        static ExitCode exitCode;
-
-        public MySampleServer(bool _autoAccept, int _stopTimeout)
+        public SampleServer Server { get; private set; }
+        public Task Status { get; private set; }
+        public DateTime LastEventTime { get; private set; }
+        public int ServerRunTime { get; private set; } = Timeout.Infinite;
+        public static bool AutoAccept { get; private set; }
+        public static ExitCode ExitCode { get; private set; }
+        public static Uri ReverseConnectUrl { get; private set; }
+
+        public MySampleServer(bool _autoAccept, int _stopTimeout, Uri _reverseConnectUrl)
```

Comment:
>Uri _reverseConnectUrl [](start = 66, length = 22)

For the .Net Core Console Server, I would the reverseConnectUrl should be optional, maybe as default parameter (= null) or with second constructor